src/aap_eda/tasks/orchestrator.py

```diff
@@ -32,23 +37,36 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class QueueNotFoundError(Exception):
+    """Raised when a queue is not found."""
+
+    ...
+
+
 def _manage_process_job_id(process_parent_type: str, id: int) -> str:
     """Return the unique job id for the activation manager task."""
     return f"{process_parent_type}-{id}"
 
 
+def get_process_parent(
+    process_parent_type: str,
+    parent_id: int,
+) -> Union[Activation, EventStream]:
+    if process_parent_type == ProcessParentType.ACTIVATION:
+        klass = Activation
+    else:
+        klass = EventStream
```

Comment:
Possible use for the python match statement with a default of "unknown type" exception. Yes, it was pre-existing but a check that a known type was passed would be better than making a default assumption.

Reply:
Well, this is a existing code, just now I encapsulate it in a function to be reused. IRCC when we started to implement event streams, we discussed about this, the point is that it is not an assumption, but I agree that adding such a exception does't make any difference. To me that would be out of the scope of this PR, let's do it in a different one, do you agree?

Reply:
Anyway, I added the exception. ;)